release-calendar/src/client/components/EventCard.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../stylesheets/eventCard.scss';
+import * as React from 'react';
+import {ApiService} from '../api-service';
+import {Channel} from '../../types';
+import {EventApi} from '@fullcalendar/core';
+import {ReleaseDates} from '../models/view-models';
+import moment from 'moment-timezone';
+
+export interface EventCardProps {
+  eventApi: EventApi;
+}
+
+interface EventCardState {
+  releaseDates: ReleaseDates;
+}
+
+export class EventCard extends React.Component<EventCardProps, EventCardState> {
```

Comment:
Can this component be renamed to something specific to Releases? All the components are named that way, but `EventCard` is more FullCalendar-esque.

Reply:
ah, ReleaseCard?

Reply:
that works